### ISSUE TITLE
Add cochleagram and "auditory" DNN to feature extractors

### ISSUE THREAD

**Issue by PeerHerholz** [neuroscout/neuroscout#784]
Ahoi hoi gang,

within our discussion wrt auditory features and things (e.g. #702 and #596), I wanted to propose the inclusion of the cochleagram feature (standard version [here](https://github.com/mcdermottLab/pycochleagram) and tensorflow version [here](https://github.com/jenellefeather/tfcochleagram)) which could act as a new, adjustable low-level feature. This would also allow us to integrate DNN layer based features from this [pre-trained network](https://github.com/mcdermottLab/kelletal2018). This could be utilized for both music and speech (looking in your direction @rbroc). For references please see [here](http://mcdermottlab.mit.edu/papers/Kell_etal_2018_DNN_auditory_cortex.pdf).

**Comment by adelavega** [comment]
hey @PeerHerholz circling back to this. If we're going to include this, let's first include it into pliers.

Any advice on "standard" vs tf version? I assume only the latter would work for the pre-trained network?

Let's get the ball rolling on some of these features! Want to meet sometime soon?

**Comment by PeerHerholz** [comment]
Yeah, sure thing, thx for getting back to this. I didn't compare the `standard` and `tf` version directly, I used the `standard` version a while back and it also worked with the `pre-trained network`.  Next week is so far pretty free for me.

**Comment by adelavega** [comment]
Hey, next week would be better actually, I'm moving this week. @rbroc, want to chat about this too?

**Comment by PeerHerholz** [comment]
Sure thing, just let me know!

**Comment by rbroc** [comment]
Would be great! Next week works for me too (best from Weds on, but earlier on can also work).
